### PR TITLE
POD13-65 polyn.gen.release task

### PR DESCRIPTION
**Pull request by brandynbennett**
Adds a `mix polyn.gen.release` task to create or inject (phoenix apps also create this) a `release.ex` file that includes a `polyn_migrate` function releases can use to execute migrations. Will allow for multiple otp_apps within the same project to execute migrations as pointed out [here](https://github.com/SpiffInc/polyn/pull/29)

### CODE REVIEW

File: polyn_elixir_client/README.md [hunk 1]
```diff
@@ -122,14 +114,35 @@ Polyn uses a shared Key-Value bucket in NATS to avoid re-running migrations. It
...
+defmodule MyApp.Release do
+  @app :my_app
+
+  def polyn_migrate do
+    load_app()
+    {:ok, _apps} = Application.ensure_all_started(:polyn)
 
-You can use it from the release like this:
+    dir = Path.join([:code.priv_dir(@app), "polyn", "migrations"])
+    Polyn.Migration.Migrator.run(migrations_dir: dir)
```
Comment (mmmries) [inline polyn_elixir_client/README.md:128]: ❤️ 💛 💙 💚 💜 I love that the API has become as simple as "run all the migrations in this directory". This simple out-of-the-box process will be easy for people to adopt, but also if they want to apply migrations after boot, or some other interesting use-case, they just need to find their directory and call this function 🙌

File: polyn_elixir_client/lib/mix/tasks/polyn.gen.release.ex [hunk 2]
```diff
@@ -0,0 +1,98 @@
+defmodule Mix.Tasks.Polyn.Gen.Release do
```
Comment (mmmries) [inline polyn_elixir_client/lib/mix/tasks/polyn.gen.release.ex:1]: I think this is a handy tool for people just starting out, but we I would be a bit cautious if people start asking for feature requests about the way we create/inject the `release.exs` file. I think we should just support this basic version here and then if they want anything more, we should just give people documentation that shows the basic version of the function they can add to whatever release process they are using.

**Review by mmmries — APPROVED** [review]
:shipit: :shipit: :shipit: :shipit: :shipit: :shipit:

File: polyn_elixir_client/lib/mix/tasks/polyn.gen.release.ex (hunk 2 above)
Comment (brandynbennett) [reply polyn_elixir_client/lib/mix/tasks/polyn.gen.release.ex:1]: @mmmries That makes sense. I think that's a good boundary to set.

**Pull request merged**